automation/deploy-bin.ts

```diff
@@ -122,21 +49,6 @@ export async function createGitHubRelease() {
  * the files.
  */
 export async function release() {
-	console.log(`Creating release assets for CLI ${version}`);
-	try {
-		await zipStandaloneInstaller();
-	} catch (error) {
-		console.log(`Error creating standalone installer zip file: ${error}`);
-		process.exit(1);
-	}
-	if (process.platform === 'win32' || process.platform === 'darwin') {
-		if (await fs.pathExists(oclifInstallers[process.platform])) {
-			await fs.rename(
-				oclifInstallers[process.platform],
-				renamedOclifInstallers[process.platform],
-			);
-		}
-	}
 	try {
 		await createGitHubRelease();
```

Comment:
Aren't we creating the release from the CI in https://github.com/balena-io/resin-concourse/blob/node-cli/pipelines/github-events/template.yml#L912-L916 ?

Reply:
Yes! This function is only called by Travis or Appveyor via the `npm release` command. Balena CI calls `npm package`, which does not call `createGitHubRelease()`. It should be OK to delete the whole `deploy-bin.ts` file (as it currently stands) when we permanently switch off Travis and Appveyor.